### DISCUSSION TITLE
New release of tracing-error?

### DISCUSSION THREAD

**Discussion by korrat** [tokio-rs/tracing#1680] [General]
The latest released `tracing-error` 0.1.2, which was released March 2020, still depends on `tracing-subscriber` 0.2.25 and does not build with `tracing-subscriber` 0.3. Could you release a new version of `tracing-error` that contains the updated dependency?

**Comment by hawkw** [comment]
I'm planning to release it today, sorry for the delay!

**Reply by hawkw** [reply]
https://crates.io/crates/tracing-error/0.2.0 :)

**Reply by korrat** [reply]
Thanks a lot!
